[BUG] Cloudfront invalidation error on full release

missing trailing quote showed up on 2.3 plublis:

```
[FAIL] Could NOT publish to documentation site
    /bin/sh: 1: Syntax error: Unterminated quoted string
Traceback (most recent call last):
  File "/usr/share/miniconda3/envs/bk-release-deploy/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/share/miniconda3/envs/bk-release-deploy/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/runner/work/bokeh/bokeh/release/__main__.py", line 35, in <module>
    steps.execute()
  File "/home/runner/work/bokeh/bokeh/release/pipeline.py", line 59, in execute
    self._system.abort()
  File "/home/runner/work/bokeh/bokeh/release/system.py", line 51, in abort
    raise RuntimeError()
RuntimeError
Error: Process completed with exit code 1.
```

--- a/release/deploy.py
+++ b/release/deploy.py
@@ -57,7 +57,7 @@ def publish_documentation(config: Config, system: System) -> ActionReturn:
             version = config.version
             system.run(f"aws s3 sync {path} s3://docs.bokeh.org/en/latest/ {flags}")
             system.run(f"aws s3 sync {path} s3://docs.bokeh.org/en/{version}/ {flags}")
-            system.run(f'aws cloudfront create-invalidation --distribution-id {CLOUDFRONT_ID} --paths "/en/latest*" "/en/{version}*')
+            system.run(f'aws cloudfront create-invalidation --distribution-id {CLOUDFRONT_ID} --paths "/en/latest*" "/en/{version}*"')
         return PASSED("Publish to documentation site succeeded")
     except RuntimeError as e:
         return FAILED("Could NOT publish to documentation site", details=e.args)

